<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-AVN7JFN" ignored="0" name="savings" tests="6" failures="1" timestamp="16 Mar 2020 02:16:58 GMT" time="0.114" errors="0">
  <testcase name="assertEquals" time="0.027" classname="com.mycompany.app.AssertGame">
    <failure type="java.lang.AssertionError" message="expected [This assertion will fail] but found [assertion]">
      <![CDATA[java.lang.AssertionError: expected [This assertion will fail] but found [assertion]
at com.mycompany.app.AssertGame.assertEquals(AssertGame.java:14)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- assertEquals -->
  <testcase name="assertFalse" time="0.002" classname="com.mycompany.app.AssertGame"/>
  <testcase name="assertNotEquals" time="0.003" classname="com.mycompany.app.AssertGame"/>
  <testcase name="assertNotNull" time="0.002" classname="com.mycompany.app.AssertGame"/>
  <testcase name="assertNull" time="0.014" classname="com.mycompany.app.AssertGame"/>
  <testcase name="assertTrue" time="0.003" classname="com.mycompany.app.AssertGame"/>
</testsuite> <!-- savings -->
